.github/workflows/deploy-java-docs.yml

```diff
@@ -0,0 +1,36 @@
+    name: GitHub Pages Deploy Action
+    on:
+        # Branch name(s) which have the source file for site/documentation
+        push:
+            branches:
+                - "*"
```

Comment:
In my opinion, you should avoid using wild cards and only update API documentation whenever there is a push in default branches (main or develop)

Reply:
fixed